### PR TITLE
FEED-5785: add maxWaitTime and waitForNavTimeout to structures

### PR DESCRIPTION
**Pull request by sangn-splunk**
Resolves https://splunk.atlassian.net/browse/FEED-5785

----

### Before the change?

* The documentation did not show `maxWaitTime` and `waitForNavTimeout`. 

image

### After the change?

* The documentation show now show `maxWaitTime` and `waitForNavTimeout`. 

### Pull request checklist 
- [ ] Acceptance Tests have been updated, run (`make testacc`), and pasted in this PR (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

#### Acceptance Test Output

```

```

### Does this introduce a breaking change?

- [ ] Yes
- [ ] No

----

### CODE REVIEW

**Review by greatestusername-splunk — CHANGES_REQUESTED** [review]
Please update the test in `resource_browser_check_v2_test.go` to include these attributes and  `TestCheckResourceAttr` entries for each of them so they are included in the functional test? Thanks!

File: synthetics/structures.go [hunk 1]
```diff
@@ -963,6 +963,14 @@ func flattenStepsData(checkSteps *[]sc2.StepsV2) []interface{} {
 
 			cl["wait_for_nav"] = checkStep.WaitForNav
 
+			if checkStep.WaitForNavTimeout != 0 {
+				cl["wait_for_nav"] = checkStep.WaitForNavTimeout
```
Comment (greatestusername-splunk) [inline synthetics/structures.go:967]: Think "wait_for_nav" collides with another property here

**Review by greatestusername-splunk — DISMISSED** [review]

File: synthetics/structures.go (hunk 1 above)
Comment (sangn-splunk) [reply synthetics/structures.go]: Fixed collision in 6ff88ca

**Review by bchhay-splunk — APPROVED** [review]
LGTM. thanks for fixing this!

**Pull request merged**